src/PrimDist.hs

```diff
@@ -226,9 +233,9 @@ prob ::
   -> Double     -- ^ density
 prob (DirichletDist xs) ys
   | Prelude.sum xs' /= 1 = error "dirichlet can't normalize"
-  | otherwise 
+  | otherwise
     = case dirichletDistribution (UV.fromList xs')
-      of Left e -> error "dirichlet error"
+      of Left e  -> error "dirichlet error"
```

Comment:
I think this formatting is ugly, but I'll wait until the project has been auto-formatted before pushing it